### ISSUE TITLE
Uncaught TypeError: undefined is not a function

### ISSUE THREAD

**Issue by RobTrew** [FoldingText/foldingtext-for-atom#29]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 0.202.0
**System**: Mac OS X 10.10.3
**Thrown From**: [foldingtext-for-atom](https://github.com/FoldingText/foldingtext-for-atom) package, v0.3.0
### Stack Trace

Uncaught TypeError: undefined is not a function

```
At /Users/robintrew/.atom/packages/foldingtext-for-atom/lib/extensions/item-path-grammar.coffee:23

TypeError: undefined is not a function
    at ItemPathGrammar.module.exports.ItemPathGrammar.tokenizeLine (/Users/robintrew/.atom/packages/foldingtext-for-atom/lib/extensions/item-path-grammar.coffee:33:26)
    at TokenizedBuffer.module.exports.TokenizedBuffer.buildTokenizedLineForRowWithText (/Applications/Atom.app/Contents/Resources/app.asar/src/tokenized-buffer.js:520:28)
    at TokenizedBuffer.module.exports.TokenizedBuffer.buildTokenizedLineForRow (/Applications/Atom.app/Contents/Resources/app.asar/src/tokenized-buffer.js:506:19)
    at TokenizedBuffer.module.exports.TokenizedBuffer.tokenizeNextChunk (/Applications/Atom.app/Contents/Resources/app.asar/src/tokenized-buffer.js:286:43)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/tokenized-buffer.js:263:26
    at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/underscore-plus/node_modules/underscore/underscore.js:666:47
```
### Commands

```
```
### Config

``` json
{
  "core": {
    "disabledPackages": [
      "atom-language-xquery",
      "language-purescript",
      "language-haskell"
    ],
    "themes": [
      "atom-dark-ui",
      "atom-light-syntax"
    ]
  },
  "foldingtext-for-atom": {}
}
```
### Installed Packages

``` coffee
# User
foldingtext-for-atom, v0.3.0
foldingtext-markdown, v1.3.0
linter, v0.12.6
linter-coffeelint, v0.2.4
minimap, v4.9.0

# Dev
No dev packages
```

**Comment by jessegrosjean** [comment]
This is caused by the latest Atom 0.202.0 update. I was using some private API and it changed. Will get a fix soon.

**Comment by jessegrosjean** [comment]
This should be fixed in FoldingText 0.3.1
